.travis.yml

```
@@ -0,0 +1,20 @@
+services:
+- docker
+script:
+  # Build
+  - docker build --target test --tag todo-app:test .
+
+  #Tests
+  #Unit & Integration tests
+  - docker run todo-app:test tests
+
+  # End to End test
+  - docker run -e SECRET_KEY=$SECRET_KEY -e TOKEN=$TOKEN todo-app:test tests_e2e
```

Really minor, but we can take advantage of a short-syntax here by switching to:
```suggestion
  - docker run -e SECRET_KEY -e TOKEN todo-app:test tests_e2e
```

---

Done